spec/controllers/student_controller_spec.rb

```diff
@@ -10,7 +10,7 @@
       it 'populates an array of all students' do
         students = [create(:student)]
         get :index
-        expect(assigns(:students)).to eq(students)
+        expect([assigns(:students).last]).to eq(students)
```

Comment:
Looks like the test env db and the dev env db are tainting each other.. or maybe are the same one? We should fix that instead of doing this here, I think. I'll look into it.